projects/aws/amazon-eks-pod-identity-webhook/Makefile

```diff
@@ -0,0 +1,68 @@
+REPO?=amazon-eks-pod-identity-webhook
+CLONE_URL?=https://github.com/aws/$(REPO).git
+GIT_COMMIT?=$(shell cat GIT_COMMIT)
+
+ifeq ("$(CLONE_URL)","")
+	$(error No clone url was provided.)
+endif
+
+ifeq ("$(REPO)","")
+	$(error No repository name was provided.)
+endif
+
+ifeq ("$(GIT_COMMIT)","")
```

Comment:
would this default off the GIT_COMMIT file?

Reply:
Yup